docs/nextflow_schema/sample_sheet_schema_specification.md

```diff
@@ -94,4 +94,4 @@ For example:
 ```
 
 will convert the `field` value to a meta value, resulting in the channel `[[id:value]...]`
-See [here](https://github.com/nextflow-io/nf-schemab/ce3aef60e5103ea4798375fe6c59bae41b7d2a25/plugins/nf-scheschemctResources/schema_input.json#L10-L25) for an example in the sample sheet.
+See [here](https://github.com/nextflow-io/nf-schema/blob/ce3aef60e5103ea4798375fe6c59bae41b7d2a25/plugins/nf-schema/src/testResources/schema_input.json#L10-L25) for an example in the sample sheet.
```

Comment:
Ooft I just checked and realised that because this is pointing to a branch/old commit that the path needs to actually be:
```
https://github.com/nextflow-io/nf-schema/blob/ce3aef60e5103ea4798375fe6c59bae41b7d2a25/plugins/nf-validation/src/testResources/schema_input.json#L10-L25
```

This is because the code was still in the `nf-validation` folder at that commit. 
Perhaps the whole link should be updated to a newer commit (post this MR)?

Reply:
Might also be worth checking for any other links which point to fixed commits and updating those too?

Reply:
For this example, we can link to master: https://github.com/nextflow-io/nf-schema/blob/master/plugins/nf-validation/src/testResources/schema_input.json#L10-L24